cmd/containerd/main.go

```diff
@@ -120,11 +120,13 @@ high performance container runtime
 		)
 		switch runtime {
 		case "runc":
+			logrus.Info("containerd: run with runc executor")
```

Comment:
My intent was not to change the format of logrus output.
I just thought a bunch of the `"containerd:"` strings can be simplified to a single literal.
Of course, such a simplification can be done in a separate PR in the future. (Probably in Phase Two or later?)
So the PR looks good to me as-is.